diff --git a/node_modules/jsx-ast-utils/.babelrc b/node_modules/jsx-ast-utils/.babelrc
deleted file mode 100644
index 4f57458..0000000
--- a/node_modules/jsx-ast-utils/.babelrc
+++ /dev/null
@@ -1,8 +0,0 @@
-{
-  "presets": ["env"],
-  "plugins": [
-    ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
-    "transform-object-rest-spread",
-    // "@babel/plugin-proposal-nullish-coalescing-operator", // TODO: update to babel 7
-  ],
-}
